packages/error-reporting/test/system-test/testAuthClient.js

```diff
@@ -0,0 +1,353 @@
+/**
```

Comment:
Can you move this directory up a level, so that it matches our others:

```
/
  /packages
    /error-reporting
      /src
      /system-test (use the real API/environment)
      /test (unit tests, no API endpoints hit)  
```